src/dpp/message.cpp

```diff
@@ -882,6 +882,38 @@ bool attachment::is_remix() const {
 	return flags & a_is_remix;
 }
 
+time_t attachment::get_expire_time() const {
+	size_t attr_position = url.find('?');
```

Comment:
This can be `const`

Reply:
which we don't enforce

Reply:
Yeah, generally i do not use const in local variables (no particular reason, just forgetting to use it). I can add it if needed

Reply:
> Yeah, generally i do not use const in local variables (no particular reason, just forgetting to use it). I can add it if needed

No need :)